benchmark/util/text-encoder.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const common = require('../common.js');
+
+const BASE = 'string\ud801';
+
+const bench = common.createBenchmark(main, {
+  n: [1e4],
+  op: ['encode', 'encodeInto']
```

Comment:
This would be better if it incorporated a range of inputs for different sizes

Reply:
Thank you for the review. I just updated the pull request.